x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/engine_overview_empty.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { useValues } from 'kea';
+
+import { i18n } from '@kbn/i18n';
+import {
+  EuiPageHeader,
+  EuiPageHeaderSection,
+  EuiPageContent,
+  EuiPageContentHeader,
+  EuiPageContentBody,
+  EuiTitle,
+  EuiText,
+  EuiButton,
+  EuiSpacer,
+} from '@elastic/eui';
+
+import { EngineLogic } from '../engine';
+
+import { DOCS_PREFIX } from '../../routes';
+import {
+  DOCUMENT_CREATION_DESCRIPTION,
+  DOCUMENT_API_INDEXING_TITLE,
+  DOCUMENT_API_INDEXING_DESCRIPTION,
+} from '../document_creation/constants';
+// TODO
+// import { DocumentCreationButtons, CodeExample } from '../document_creation'
+
+export const EmptyEngineOverview: React.FC = () => {
+  const { engineName } = useValues(EngineLogic);
+
+  return (
+    <>
+      <EuiPageHeader>
+        <EuiPageHeaderSection>
+          <EuiTitle size="l">
+            <h1>
+              {i18n.translate('xpack.enterpriseSearch.appSearch.engine.overview.empty.heading', {
+                defaultMessage: 'Engine Setup',
+              })}
+            </h1>
+          </EuiTitle>
+        </EuiPageHeaderSection>
+        <EuiPageHeaderSection>
+          <EuiButton href={`${DOCS_PREFIX}/index.html`} target="_blank">
+            {i18n.translate(
+              'xpack.enterpriseSearch.appSearch.engine.overview.empty.headingAction',
+              { defaultMessage: 'View Documentation' }
```

Comment:
Ditto for this, probably better to not title case here

```suggestion
              { defaultMessage: 'View documentation' }
```

Reply:
What are your thoughts on the "View Analytics" and "View API Logs" buttons? (see screenshot)

<img width="589" alt="Screen Shot 2020-11-18 at 10 31 36 AM" src="https://user-images.githubusercontent.com/549407/99572284-47651f80-2989-11eb-9739-ecf8e3809fe2.png">

This is _partially_ why I left copy as-is haha, it felt like a can of worms I didn't have the expertise to go into 💦

Reply:
@chriscressman can I get your expert opinion on this one as well? Should it be "View Analytics" since Analytics is a feature title? I'm guessing yes?

Also if it's easier for you to hop in a zoom call and quickly point out copy issues to me and have me fix them on the go, please do let me know! 🙏

Reply:
https://github.com/elastic/kibana/pull/83504/commits/7c6c8c8d3937c6c081603976781e6a5d7d07c2fa